### ISSUE TITLE
User guide: update NVDA's system requirements

### ISSUE THREAD

**Issue by josephsl** [nvaccess/nvda#10923]
### Is your feature request related to a problem? Please describe.
With NVDA supporting Windows 7 and later, user guide still refers to outdated system requirements. Notably, minimum memory is listed as 256 MB, but Windows 7 and later requires at least 1 GB on 32-bit systems and 2 GB on 64-bit platforms. Along with this, storage requirements will need to be updated to account for additions and add-on installations.

### Describe the solution you'd like
Change system requirements:

* Remove memory requirement, as Windows 7 or later is assumed.
* Change storage requirement to 130 MB, along with a note that add-ons will require additional disk space.

### Describe alternatives you've considered
Leave it as is.

### Additional context
There might be other things to note when considering system requirements, but memory requirement is the most prominent one that is outdated.

Thanks.

**Comment by feerrenrut** [comment]
Since NVDA itself does not have these requirements, it might be more sensible to remove this. Any system that Windows 7 or later runs on has the minimum requirements.

**Comment by DrSooom** [comment]
See also: Issue #9967

**Comment by josephsl** [comment]
Hi, I think Reef is right in that it is time to remove memory requirement text from user guide. Thanks.

**Comment by DrSooom** [comment]
I agree – RAM and CPU can be omitted. But the free storage space must be updated to – let's say for now – 128 MB. Okay, if you have less than 128 MB of free storage, then you have tons of other problems. 😉

**Comment by josephsl** [comment]
Hi, let’s say 100 to 120 MB to make life easier for people. Thanks.

**Comment by DrSooom** [comment]
The NVDA Alpha-19852 Installer archive size is 117.91 MB. Therefore I suggest 130 MB. That should be enough for the next two years. (Personally I like 2^n numbers quite a lot.)

**Comment by josephsl** [comment]
Hi, with a caveat that more disk space will be required if installing add-ons. Thanks.

**Comment by DrSooom** [comment]
That's fully correct. But this should be handled during solving the issues #2467, #9020 and #9967.

**Comment by feerrenrut** [comment]
That all sounds reasonable. Specify: "130 MB storage* More for add-ons" @josephsl could you update the issue description please?

**Comment by Adriani90** [comment]
Counting my folder, nvda 2020 last alpha has 121 mb, + the folder in %appdata% which in my case has 26 mb, I think most people would not run into issues with at least 150 mb free space before installing NVDA.
